docs/evm/using.mdx

```diff
@@ -38,14 +38,14 @@ Manually method: Add FlowEVM as a custom network to MetaMask:
 4. Click `**Add a network manually**`
 5. In the `**Add a network manually**` dialog that appears, enter the following information for FlowEVM mainnet:
     
-| Name            | Value                                  |
-|-----------------|----------------------------------------|
-| Network Name    | FlowEVM Mainnet                        |
-| Description     | The public RPC url for FlowEVM mainnet |
-| RPC Endpoint    | https://mainnet.evm.nodes.onflow.org   |
-| Chain ID        | 666                                    |
-| Currency Symbol | FLOW                                   |
-| Block Explorer  | https://flowdiver.io                   |
+| Name            | Value                                    |
+|-----------------|------------------------------------------|
+| Network Name    | FlowEVM Crescendo                        |
```

Comment:
```suggestion
| Network Name    | FlowEVM PreviewNet                        |
```
We're using PreviewNet in other places https://developers.flow.com/evm/Build/networks

Reply:
ok, I see the network as "Crescendo PreviewNet" 
https://github.com/onflow/flow-evm-gateway?tab=readme-ov-file#flowevm-gateway-endpoints
Here as well. I'll make it consistent